cmd/osm-injector/osm-injector.go

```diff
@@ -241,6 +242,7 @@ func main() {
 	}
 
 	<-stop
+	cancel()
```

Comment:
Is there a reason this is necessary on top of the `defer cancel()` where this is created? And it looks like this was already being called at more-or-less this same point in time from within `signals.RegisterExitHandlers`.

Reply:
nope, removed that call